it/src/main/java/com/google/cloud/teleport/it/spanner/DefaultSpannerResourceManager.java

```diff
@@ -90,6 +91,9 @@ private DefaultSpannerResourceManager(Builder builder) {
     this.spanner = spanner;
     this.instanceAdminClient = spanner.getInstanceAdminClient();
     this.databaseAdminClient = spanner.getDatabaseAdminClient();
+
+    // Check that the project ID conforms to GCS standards
```

Comment:
GCP standards?

Reply:
I had Google Cloud Storage in my head for some reason. Fixed, thanks.